backend/import/do_import.py

```diff
@@ -0,0 +1,157 @@
+#
+# Copyright (c) 2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+import os
+import json
+from multiprocessing.dummy import Pool
+
+import mo_api
+
+
+def get_orgunit_data(uuid):
+    """Get the data we need to display for this particular org. func."""
+    ou = mo_api.MOOrgUnit(uuid)
+    # Parent - UUID if exists, ROOT if not.
+    parent = ou.json['parent']['uuid'] if ou.json['parent'] else 'ROOT'
+    # For employees, we need job function, name and UUID.
+    employees = [
+        (
+            e['person']['name'], e['person']['uuid'], e['job_function']['name']
+        ) for e in ou.engagement
+    ]
+    # For associates: Association type, job function, name and UUID.
+    associates = [
```

Comment:
Verbage: an associate is a person, whereas what we represent as `association` is not.

Reply:
... but an association seems to always contain one person in the current data set? Or is that a simplification?

Reply:
The people referred to here are listed as "tilknyttede" in the GUI. 

The people referred to in "engament" are listed as "medarbejdere".

Reply:
[<img width="100%" alt="definition of associate on Merriam-Webster" title="definition of associate on Merriam-Webster" src="https://user-images.githubusercontent.com/59300/48255268-87917f00-e40c-11e8-8d5f-9da652a798f5.png">](https://www.merriam-webster.com/dictionary/associate)

Reply:
I'd be happy if you called it “associated” instead — “associates” just feels inaccurrate 🙂